csvViewer/reader.rb

```diff
@@ -0,0 +1,9 @@
+require 'csv'
+module CSVFILE
```

Comment:
```suggestion
class CSVFILE
```
no need to use module, just a class